views/headlines.hbs

```diff
@@ -64,15 +64,15 @@
 										stroke-miterlimit="10" />
 								</svg>
 							</div>
-
+							
 							{{#each data}}
 							<div class="card">
 								<h4 class="issue-key">
 									<img class="isssue-type-avatar" src={{fields.issuetype.iconUrl}}></img>
-									{{key}}
+									<a href={{issueLink}} target="_blank">{{key}} </a>
 								</h4>
 								<h5 class="issue-summary">{{fields.summary}}</h5>
-								<h6>Last updated: {{DateFormatter fields.updated}} </h6>
+								<h6>Last updated: {{DateFormatter fields.updated}} AGO</h6>
```

Comment:
@Farhan-Haseeb , ago should be in small letters. rest looks good.

Reply:
@waleedwaseem  Done!